pastis/matrix_generation/matrix_from_efields.py

```diff
@@ -106,7 +126,7 @@ def calculate_ref_efield(self):
         raise NotImplementedError()
 
     def calculate_ref_efield_wfs(self):
-        """ Create the attributes self.norm, self.dh_mask, self.coro_simulator and self.efield_ref. """
+        """ Create the attributes self.norm, self.dh_mask, self.wfs_simulator and self.efield_ref_wfs. """
```

Comment:
The first three attributes in this docstring list need to be deleted, as this method only requires you to calculate the reference E-field on the WFS, not a normalization factor, DH mask or simulator instance.